src/composables/multisig.ts

```diff
@@ -13,12 +31,35 @@ interface UseMultisigOptions {
   store: Store<any>
 }
 
-/**
- * This composable provides methods related to multisig operations
- */
-export function useMultisig({ store }: UseMultisigOptions) {
+const POLLING_INTERVAL = 5000;
+const MULTISIG_TRANSCATION_EXPARATION_HEIGHT = 50;
+
+const LOCAL_STORAGE_MULTISIG_KEY = `${LOCAL_STORAGE_PREFIX}_multisig`;
+
+function storeMultisigAccounts(multisig: IMultisigAccount[], networkId: string) {
+  window.localStorage
+    .setItem(`${LOCAL_STORAGE_MULTISIG_KEY}_${networkId}`, JSON.stringify(multisig));
+}
+
+function getStoredMultisigAccounts(networkId: string): IMultisigAccount[] {
+  const storedMultisig = window.localStorage.getItem(`${LOCAL_STORAGE_MULTISIG_KEY}_${networkId}`);
+  return storedMultisig ? JSON.parse(storedMultisig) : [];
+}
+
+const multisig = ref(<IMultisigAccount[]>[]);
+const activeGaAccountId = ref('');
+const isMultisigDashboard = ref(false);
```

Comment:
Shouldn't the dashboard for the multisig have different route than the plain one?

Reply:
I asked Paolo about this and he confirmed:
![image](https://user-images.githubusercontent.com/8217670/214798834-dc8f4828-b1b1-4ee8-9b65-49fad591c438.png)